Controllers/IndicatorsController.cs

```diff
@@ -121,6 +133,91 @@ await indicatorQuery.Include(x => x.Registries)
             return Ok(indicator);
         }
 
+        // GET: api/Indicators/Goal/1
+        [HttpGet("Goals/{id:long}")]
```

Comment:
`api/Indicators/Goals/1` Means 1st goals of all indicators or the goal number 1 in indicators.
Should have been `api/Indicators/{id:long}/Goals`, meaning goals of indicator 1.